.gitignore

```diff
@@ -55,3 +55,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# pyvenv
+venv
```

Comment:
Oh, I do all my python development in virtual environments. Basically
```
pyvenv --clear venv
source venv/bin/activate
pip install --upgrade pip
pip install --upgrade setuptools
python setup.py develop
```
gets me an encapsulated development environment, then
```
python setup.py test
```
installs test_requires (not currently defined in setup.py) and runs tests.